README.md

```diff
@@ -1,33 +1,43 @@
 # Python bindings for Nix using CFFI
-Maintainer status: maintained, experimental
-Compatibility: Requires https://github.com/NixOS/nix/pull/8699
-
 
-# Getting started
-```shell
-$ nix develop tweag/python-nix#env
-$ python
->>> import nix
->>> nix.eval("1+1")
-<Nix: 2>
->>>
-```
-
-# Hacking
-```shell
-$ nix develop .#
-$ cd src
-$ python buildFFI.py
-$ python
->>> import nix
-```
```

Comment:
Why are you removing it? 
It's good that you added instructions about how to install it with pip, but I'd keep the instructions about how to run the project with Nix.

Reply:
My expectation is that most people will want to run it inside their project, not via the REPL. I thought the instructions could be confusing.
I've put them back again in the README :+1:

Reply:
Thanks.

At this point, we are still experimenting. Maybe in the end we decide to remove those instructions, but for now, I think the more information we have, the better.